warehouse/accounts/views.py

```diff
@@ -206,17 +208,28 @@ def profile(user, request):
     )
 
     # Construct the list of projects with their latest releases from query results
-    projects = [
-        {
+    archived_projects = []
+    live_projects = []
+
+    for row in request.db.execute(stmt):
+        project = {
             "name": row.name,
             "normalized_name": row.normalized_name,
+            "lifecycle_status": row.lifecycle_status,
             "created": row.created,
             "summary": row.summary,
         }
-        for row in request.db.execute(stmt)
-    ]
 
-    return {"user": user, "projects": projects}
+        if row.lifecycle_status == LifecycleStatus.Archived:
+            archived_projects.append(project)
+        else:
+            live_projects.append(project)
+
+    return {
+        "user": user,
+        "projects": live_projects,
```

Comment:
question: considering the only variable use in the template of `projects` is now to combine to project_count and to iterate over it, any reason to not have consistency and rename it `live_projects` to reduce cognitive load?

Reply:
Yeah, good call. I didn't rename it because I didn't want to churn a template variable but it's definitely confusing as-is!